package.json

```diff
@@ -280,6 +280,22 @@
           "@fluentui/eslint-plugin",
           "stylis"
         ]
+      },
+      {
+        "packages": [
```

Comment:
@Hotell can you please help me out with the best way to do this without completely killing single version policy ?

Reply:
ha, run into this as well!

I'd suggest to do following for now https://github.com/microsoft/fluentui/pull/18808/files#r665234382